pkg/execute/actions.go

```diff
@@ -0,0 +1,56 @@
+package execute
+
+import (
+	"bytes"
+	"fmt"
+	"text/tabwriter"
+
+	"github.com/kubeshop/botkube/pkg/config"
+)
+
+type ActionManager struct {
+	actions map[string]bool
+}
+
+func NewActionManager(cfg config.Actions) *ActionManager {
+	a := make(map[string]bool)
+	for k, v := range cfg {
+		a[k] = v.Enabled
+	}
+	return &ActionManager{a}
+}
+
+func (a *ActionManager) listActions() map[string]bool {
+	out := make(map[string]bool)
+	for k, v := range a.actions {
+		out[k] = v
+	}
+	return out
+}
+
+func (a *ActionManager) tabularOutput() string {
+	buf := new(bytes.Buffer)
+	w := tabwriter.NewWriter(buf, 5, 0, 1, ' ', 0)
+	fmt.Fprintln(w, "ACTION\tENABLED")
+	for name, enabled := range a.listActions() {
+		fmt.Fprintf(w, "%s\t%v\n", name, enabled)
+	}
+	w.Flush()
+	return buf.String()
+}
+
+func (a *ActionManager) enableAction(name string) bool {
```

Comment:
BTW, why do we need such methods `enableAction` and `disableAction`? We don't need to write the config to the `actions map[string]bool` map, as Botkube needs to be restarted to apply the changes.

So using `PersistActionEnabled` is totally sufficient.

Remember, that filters work a bit differently - we can enable/disable them in runtime without restarting app. With actions, we'll need to restart the whole app, so we don't need to change anything in runtime.

Reply:
I ended up deleting these functions.